tests/test_basic.py

```diff
@@ -44,6 +60,22 @@ def test_failed_jobs(self):
         response = self.client.get('/failed')
         self.assertEqual(response.status_code, HTTP_OK)
 
+    def test_worker_python_version_field(self):
+        w = Worker(['q'])
+        w.register_birth()
+        response = self.client.get('/workers.json')
+        data = json.loads(response.data.decode('utf8'))
+        if getattr(w, 'python_version', None):
+            self.assertEqual(w.python_version, data['workers'][0]['python_version'])
```

Comment:
It makes sense to add else branch in both tests to standardise behaviour when there is no python/rq version in worker.

Reply:
Yes, I don’t know why I missed it. Edited.